src/components/reward-item-filter/resources/RewardItemApi.ts

```diff
@@ -0,0 +1,31 @@
+import {GraphQLClient} from "graphql-request";
+import {endpoint} from "@/api/TarkovDevApi.ts";
+import {RewardItemResponse} from "./RewardItemResponse.ts";
+
+export class RewardItemApi {
```

Comment:
This should be moved to `src/api/TarkovDevApi.ts`. Alternatively, if we want it more granularly, it should at least be moved to the `api` folder.